### DISCUSSION TITLE
Show tab names?

### DISCUSSION THREAD

**Discussion by YousufSSyed** [sxyazi/yazi#1767] [1. Q&A]
Right now tabs are numbered 1 2 3 ..., I would like them to have labels of their directory names, it make it easier for me to manage.

**Comment by sxyazi** [comment]
> tab_width (Number): Tab maximum width. When set to a value greater than 2, the remaining space will be filled with the tab name, which is current directory name.

https://yazi-rs.github.io/docs/configuration/theme
